src/index.js

```diff
@@ -1331,6 +1339,22 @@ class CloudManagerAPI {
     })
   }
 
+  _commerceLogTransform () {
+    const liner = new Transform({ objectMode: true })
+    liner._transform = function (data, _, done) {
+      const lines = data.toString().split('\n')
+      for (const line of lines) {
+        if (line !== '') {
+          const parsedLine = JSON.parse(line.replace('\n', '\\n'))
```

Comment:
@rwalter215 I think this should have a try/catch block surrounding it. Not sure what to do with the error, maybe just swallow it. WDYT?

Reply:
Yeah that's a really good idea. And same, not sure what else we would do but swallow it. I'll implement this change

Reply:
the only thing I can think of would be to output the line as-is, but I'd say swallow it for now and if this happens in the real world it can always be revisited.

Reply:
Sounds good, I implemented that change, should be ready to merge once all the checks finish